package.json

```diff
@@ -178,7 +180,8 @@
       "one-app-statics/package.json"
     ],
     "skip": {
-      "tag": true
+      "tag": true,
+      "changelog": true
```

Comment:
Skipping changelog as we no longer need this because of `conventional-changelog`